### DISCUSSION TITLE
Auto-paste adding an unwanted extra line before link

### DISCUSSION THREAD

**Discussion by piisamirontta** [RyotaUshio/obsidian-pdf-plus#262] [Q&A]
I am using the auto-paste function with this link format:
`- #{{color}} {{selection}} [@{{file.basename}}] ({{linkWithDisplay}})`

The idea is that the auto-paste should add a bullet point at the end of the note, no empty line before or after the point. But for some reason it seems to add an extra line break before the point, splitting the list.

I am also using the Outliner plugin. Could this interfere somehow?

**Comment by RyotaUshio** [comment]
@laineveeti hey sorry for the late response, did you try adding a line break at the end of the link format? like

```
- #{{color}} {{selection}} [@{{file.basename}}] ({{linkWithDisplay}})

```

(notice the empty second line)

I've quickly tested this, and it seems to be working. If you perform auto-paste consecutively, you will get a bullet list without a line break in it. And it worked whether the Outliner plugin is enabled or not.

https://github.com/user-attachments/assets/61c766c9-638c-4035-a670-c720798db193
